src/record.c

```diff
@@ -292,6 +293,15 @@ int janus_recorder_add_extmap(janus_recorder *recorder, int id, const char *extm
 	return 0;
 }
 
+int janus_recorder_description(janus_recorder *recorder, const char *description) {
```

Comment:
You need a

	if(!recorder)
		return -1;

as the first check here.

Reply:
Fixed